### DISCUSSION TITLE
Lazy loading images

### DISCUSSION THREAD

**Discussion by MayaHM1** [liedekef/events-made-easy#174] [General]
I tested my website performance on GTMetrix and it said that lazy loading the event image is slowing down the initial page display time and leading to poor performance. I think that `#_EVENTIMAGE` outputs an image tag with the attribute `loading="lazy"` which GTMetrix says shouldn't be used if the image is in the initial visible area. 

It's easy enough to get around this by using `<img src="#_EVENTIMAGEURL">` and this improves the page display performance significantly.

Just a suggestion but given that event pages often have the image at the top maybe it would be a good idea to make the lazy loading optional to improve performance and SEO ranking?

**Comment by liedekef** [comment]
Lazy loading is not done by EME, but by WP itself. Please read this: https://www.wpbeginner.com/wp-tutorials/how-to-properly-disable-lazy-load-in-wordpress/

**Reply by MayaHM1** [reply]
Ok thanks for the clarification!
